### DISCUSSION TITLE
gh-pages don`t work

### DISCUSSION THREAD

**Discussion by Powerhair** [community/community#40088] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I use npm gh-pages. Before a created in my repo new branch "gh-pages".
I deploydes my project to gh-pages. But how i try to github.io i have issues 404.

My repo https://github.com/Powerhair/mesto

**Comment by airtower-luna** [comment]
Your Pages configuration seems to deploy from `main` instead of the `gh-pages` branch. Try [changing the branch (step 6)](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-from-a-branch) to `gh-pages`.

**Comment by Preetiraj3697** [comment]
0


I had same issue.. Very strange issue.. My HTML was with space after title

<title>
.
.
.
<script>

Fixed, after removing space

 <title>
 <script>

or 

I had just one commit with all my files. I pushed an empty commit, refreshed the page and it worked.

git commit --allow-empty -m "Trigger rebuild"
git push
If this doesn't work, as @Hendrikto pointed out in the comments, check out the [Github status page](https://www.githubstatus.com/) and make sure GitHub Pages are operational.
